### ISSUE TITLE
ESlint crashes with syntax error when run on lower version of Node

### ISSUE THREAD

**Issue by loldrup** [eslint/eslint#10549]
When newly installed, eslint crashes, if no config file has been generated yet. Is this nominal behavior?

C:\Users\Jon\Aukitekt\frontend>npm install eslint -g
....
C:\Users\Jon\Aukitekt\frontend>eslint
C:\Users\Jon\AppData\Roaming\npm\node_modules\eslint\lib\linter.js:739
                [methodName](...args) {
                             ^^^

SyntaxError: Unexpected token ...
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:387:25)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (C:\Users\Jon\AppData\Roaming\npm\node_modules\eslint\lib\cli-engine.js:21:14)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)

**Tell us about your environment**

* **ESLint Version:**
5.0.1
* **Node Version:**
5.7.1
* **npm Version:**
3.6.0

**What parser (default, Babel-ESLint, etc.) are you using?**
default

**Please show your full configuration:**
not existing

<details>
<summary>Configuration</summary>

```js

```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js

```

```bash

```

**What did you expect to happen?**

**What actually happened? Please include the actual, raw output from ESLint.**

**Comment by platinumazure** [comment]
Hi @loldrup, thanks for the issue.

In ESLint 5.x, we don't support Node versions earlier than 6.14.0 (specifically, we support 6.14 or higher on the 6.x line, 8.10 or higher on the 8.x line, and 9.10 or higher). Please upgrade your version of the Node runtime or downgrade to ESLint 4.x. Thanks!

**Comment by loldrup** [comment]
Thanks for your answer :)
Perhaps ESlint could print that explanation, before crashing on older node versions?

**Comment by not-an-aardvark** [comment]
I think npm already provides a warning like that:

```
npm WARN engine eslint@5.0.1: wanted: {"node":"^6.14.0 || ^8.10.0 || >=9.10.0"} (current: {"node":"4.9.1","npm":"2.15.11"})
```

**Comment by platinumazure** [comment]
I would love to see a warning from ESLint myself. I have a branch up where I was working on that, but I don't know how to test it (i.e., how to mock package versions). I'll try to get back to that in the next few weeks.

**Comment by platinumazure** [comment]
This is potentially resolved by #10570, which hasn't been merged. I'll mark this as a core enhancement accordingly.
